lib/src/config/backends.rs

```diff
@@ -126,13 +127,26 @@ mod deserialization {
                 .transpose()?
                 .unwrap_or(true);
 
+            let grpc = toml
+                .remove("grpc")
+                .map(|grpc| {
+                    if let Value::Boolean(grpc) = grpc {
+                        Ok(grpc)
+                    } else {
+                        Err(BackendConfigError::InvalidGrpcEntry)
+                    }
+                })
+                .transpose()?
+                .unwrap_or(true);
```

Comment:
@acw is this turning gRPC on by default for static backends?

Reply:
crud. yes.